### ISSUE TITLE
Proposal to add parser for bluetoothctl utility

### ISSUE THREAD

**Issue by tzeikob** [kellyjonbrazil/jc#391]
This is a parser proposal for the bluetoothctl utility, where the parser should parse both controller and device entities like so:

**$ bluetoothctl show CC:32:AF:E7:6A:E6**
```
Controller CC:32:AF:E7:6A:E6 (public)
        Name: arch
        Alias: arch
        Class: 0x006c010c
        Powered: yes
        Discoverable: no
        DiscoverableTimeout: 0x000000b4
        Pairable: no
        UUID: Handsfree                 (0000111e-0000-1000-8000-00805f9b34fb)
        UUID: Audio Source              (0000110a-0000-1000-8000-00805f9b34fb)
        UUID: Audio Sink                (0000110b-0000-1000-8000-00805f9b34fb)
        UUID: PnP Information           (00001200-0000-1000-8000-00805f9b34fb)
        UUID: A/V Remote Control Target (0000110c-0000-1000-8000-00805f9b34fb)
        UUID: A/V Remote Control        (0000110e-0000-1000-8000-00805f9b34fb)
        UUID: Handsfree Audio Gateway   (0000111f-0000-1000-8000-00805f9b34fb)
        Modalias: usb:v1D6Bp0246d0542
        Discovering: yes
```

**$ bluetoothctl show CC:32:AF:E7:6A:E6 | jc --bluetoothctl | jq .**
```
[
  {
    "address": "CC:32:AF:E7:6A:E6",
    "is_public": true,
    "name": "arch",
    "alias": "arch",
    "class": "0x006c010c",
    "powered": "yes",
    "discoverable": "no",
    "discoverable_timeout": "0x000000b4",
    "pairable": "no",
    "uuids": [
      "Handsfree                 (0000111e-0000-1000-8000-00805f9b34fb)",
      "Audio Source              (0000110a-0000-1000-8000-00805f9b34fb)",
      "Audio Sink                (0000110b-0000-1000-8000-00805f9b34fb)",
      "PnP Information           (00001200-0000-1000-8000-00805f9b34fb)",
      "A/V Remote Control Target (0000110c-0000-1000-8000-00805f9b34fb)",
      "A/V Remote Control        (0000110e-0000-1000-8000-00805f9b34fb)",
      "Handsfree Audio Gateway   (0000111f-0000-1000-8000-00805f9b34fb)"
    ],
    "modalias": "usb:v1D6Bp0246d0542",
    "discovering": "yes"
  }
]
```

**$ bluetoothctl list**
```
Controller CC:32:AF:E7:6A:E6 arch [default]
Controller AA:12:EF:E7:6A:E6 logi
```

**$ bluetoothctl list | jc --bluetoothctl | jq .**
```
[
  {
    "address": "CC:32:AF:E7:6A:E6",
    "is_default": true,
    "name": "arch"
  },
  {
    "address": "AA:12:EF:E7:6A:E6",
    "name": "logi"
  }
]
```

**$ bluetoothctl info BB:06:EF:62:B3:19**

```
Device BB:06:EF:62:B3:19 (public)
        Name: TaoTronics TT-BH026
        Alias: TaoTronics TT-BH026
        Class: 0x00240404
        Icon: audio-headset
        Paired: no
        Bonded: no
        Trusted: no
        Blocked: no
        Connected: no
        LegacyPairing: no
        UUID: Advanced Audio Distribu.. (0000110d-0000-1000-8000-00805f9b34fb)
        UUID: Audio Sink                (0000110b-0000-1000-8000-00805f9b34fb)
        UUID: A/V Remote Control        (0000110e-0000-1000-8000-00805f9b34fb)
        UUID: A/V Remote Control Cont.. (0000110f-0000-1000-8000-00805f9b34fb)
        UUID: Handsfree                 (0000111e-0000-1000-8000-00805f9b34fb)
        UUID: Headset                   (00001108-0000-1000-8000-00805f9b34fb)
        UUID: Headset HS                (00001131-0000-1000-8000-00805f9b34fb)
        RSSI: -56
        TxPower: 4
```

**$ bluetoothctl info BB:06:EF:62:B3:19 | jc --bluetoothctl | jq .**
```
[
  {
    "address": "BB:06:EF:62:B3:19",
    "is_public": true,
    "name": "TaoTronics TT-BH026",
    "alias": "TaoTronics TT-BH026",
    "class": "0x00240404",
    "icon": "audio-headset",
    "paired": "no",
    "bonded": "no",
    "trusted": "no",
    "blocked": "no",
    "connected": "no",
    "legacy_pairing": "no",
    "uuids": [
      "Advanced Audio Distribu.. (0000110d-0000-1000-8000-00805f9b34fb)",
      "Audio Sink                (0000110b-0000-1000-8000-00805f9b34fb)",
      "A/V Remote Control        (0000110e-0000-1000-8000-00805f9b34fb)",
      "A/V Remote Control Cont.. (0000110f-0000-1000-8000-00805f9b34fb)",
      "Handsfree                 (0000111e-0000-1000-8000-00805f9b34fb)",
      "Headset                   (00001108-0000-1000-8000-00805f9b34fb)",
      "Headset HS                (00001131-0000-1000-8000-00805f9b34fb)"
    ],
    "rssi": "-66",
    "txpower": "4"
  }
]
```

**$ bluetoothctl devices**
```
Device EB:66:EF:62:B3:19 TaoTronics TT-BH026
Device GG:5F:EA:F8:E5:A1 Wacom
```

**$ bluetoothctl devices | jc --bluetoothctl | jq .**
```
[
  {
    "address": "EB:66:EF:62:B3:19",
    "name": "TaoTronics TT-BH026"
  },
  {
    "address": "GG:5F:EA:F8:E5:A1",
    "name": "Wacom"
  }
]
```

**Comment by kellyjonbrazil** [comment]
Thank you for the contribution! I'll take a closer look this weekend.

**Comment by kellyjonbrazil** [comment]
PR merged into `dev`. This will make it in the next release. Thanks for the contribution!

**Comment by kellyjonbrazil** [comment]
Released in v1.23.2
